### PR TITLE
Allow usage of Context/Resources

### PR DESCRIPTION
**Pull request by tuxmea**

### CODE REVIEW

**Comment by tuxmea** [comment]
Still waiting for approval. This code is in use at a customer and works flawlessly.

File: manifests/config/context/resources.pp
Comment (david22swan) [inline manifests/config/context/resources.pp:33]: Unsure why this bit of code is necessary?
Since `$resources_name` defaults to `$name` if not given won't this always be true?

**Review by david22swan — APPROVED** [review]
LGTM

One small comment but noting that would stop it being merged.
Apologies for the wait on this, it just slipped past our notice.

**Pull request merged**
